src/modules/fancyzones/lib/util.h

```diff
@@ -2,178 +2,192 @@
 
 #include "gdiplus.h"
 
-struct Rect
+namespace FancyZonesUtils
 {
-    Rect() {}
-
-    Rect(RECT rect) :
-        m_rect(rect)
+    // Window properties relevant to FancyZones
+    struct FancyZonesWindowInfo
     {
-    }
+        // True if from the styles the window looks like a standard window
+        bool standard_window = false;
+        // True if the window is a top-level window that does not have a visible owner
+        bool no_visible_owner = false;
```

Comment:
Not that important, but we have upper camel case across all fancyzones, we might want to have same thing in this `util.cpp`.

Reply:
Good point. I'll prepare new PR with just that - restyling util.[h|cpp]

Reply:
Ah, it's just this one struct.. I'll do it here